dsl_parser/elements/policies.py

```diff
@@ -332,13 +355,19 @@ def calculate_provided(self, groups, node_templates, **kwargs):
         # the parsed value of "policies" which is only calculated in "parse"
         self._validate_and_update_groups(scaling_groups, node_templates)
         return {
-            'scaling_groups': scaling_groups
+            constants.SCALING_GROUPS: scaling_groups
         }
 
     def _create_scaling_groups(self, groups):
+        def find_scaling_policies(current_policies):
```

Comment:
Again, why create a function when you could just use list comprehension and a comment?